.grabl/automation.yml

```diff
@@ -28,19 +28,16 @@ build:
       command: |
         bazel build //...
         bazel run @vaticle_dependencies//tool/checkstyle:test-coverage
-        bazel test $(bazel query 'kind(checkstyle_test, //...)') --test_output=streamed
+        bazel test $(bazel query 'kind(checkstyle_test, //...)')
```

Comment:
Checkstyle tests shouldn't very intense so we can arrive at a result faster without this.